jython_cli.java

```diff
@@ -0,0 +1,212 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+
+//DEPS dev.jbang:jash:0.0.3
+//DEPS org.tomlj:tomlj:1.1.1
+//DEPS org.python:jython-standalone:2.7.4
+//JAVA 21
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Stream;
+import dev.jbang.jash.Jash;
+import org.tomlj.Toml;
+import org.tomlj.TomlParseResult;
+import org.tomlj.TomlTable;
+
+import org.python.util.jython;
+
+public class jython_cli {
+
+    // Increment this number with each new release of jython-cli
+    // But if the version number of Jython changes (for example 2.7.4 becomes 2.7.5), then
+    // the FIX_NUMBER is reset to 0 again.
+    private static final int FIX_NUMBER = 0;  
+
+    private static final String textJythonApp = """
+            import org.python.util.PythonInterpreter;
+            import java.util.Base64;
+            
+            public class __CLASSNAME__ {
+            
+                public static String mainScriptTextBase64 = "__MAIN_SCRIPT__";
+ 
+                public static void main(String... args) {
+                    String mainScriptFilename = "__MAIN_SCRIPT_FILENAME__";
+                    String mainScript = "";
+                    String pythonArgsScript = "";
+                    for (String arg: args) {
+                        if (pythonArgsScript.length() == 0) {
+                            if (!arg.equals(mainScriptFilename)) {
+                                pythonArgsScript += "'" + mainScriptFilename + "', ";
+                            }
+                        } else {
+                            pythonArgsScript += ", ";
+                        }
+                        pythonArgsScript += "'" + arg + "'";
+                    }
+                    if (pythonArgsScript.length() == 0) {
+                        pythonArgsScript = "'" + mainScriptFilename + "'";
+                    }
+                    pythonArgsScript = "import sys; sys.argv = [" + pythonArgsScript + "]";
+                    {
+                        byte[] decodedBytes = Base64.getDecoder().decode(mainScriptTextBase64);
+                        String text = new String(decodedBytes);
+                        mainScript = text;
+                    }
+                    {
+                        // create Python interpreter object
+                        PythonInterpreter pyInterp = new PythonInterpreter();
+                        // initialize command-line args
+                        pyInterp.exec(pythonArgsScript);
+                        // run script
+                        pyInterp.exec(mainScript);
+                    }
+                }
+            }            
+            """;
+
+    public static void main(String[] args) throws IOException {
+        List<String> deps = new ArrayList<>();
+        String jythonVersion = "2.7.4";
+        String javaVersion = "21";
+        String javaRuntimeOptions = "";
+        String jbangIntegrations = "true";
+        String ls = System.lineSeparator();
```

Comment:
I don't usually have any problems on Windows just using `\n` for lines. It's not like we're reading the text with notepad.

Reply:
Fixed.